tests/test_compositing.py

```diff
@@ -93,22 +93,44 @@ def test_concatenate_floating_point(util):
     concat.write_videofile(os.path.join(util.TMP_DIR, "concat.mp4"), preset="ultrafast")
 
 
+# def test_blit_with_opacity():
```

Comment:
I can't really understand this test nor its regression, as it's not affected by my new methods .  It was failing  like this 

```
E       AssertionError: Expected (7f,7f,00) but got (00,80,80) at timestamp 0.5
E       assert not True
```

which is the same error that I get  in **master** if I apply this diff to the test 

```diff
(lab) tin@morocha:~/lab/moviepy$ git diff
diff --git a/tests/test_compositing.py b/tests/test_compositing.py
index 5824034..d3bdee2 100644
--- a/tests/test_compositing.py
+++ b/tests/test_compositing.py
@@ -105,6 +105,7 @@ def test_blit_with_opacity():
         .with_opacity(0.5)
     )
     composite = CompositeVideoClip([clip1, clip2])
+    composite.write_videofile("compo.mp4")
     bt = ClipPixelTest(composite)
 
     bt.expect_color_at(0.5, (0x7F, 0x7F, 0x00))
```

ie,write de composed videofile before the assert, it shouldn't affect, right?  what's happening? 

As an alternative, I rewrote the test as I understood below using ColorClip. Is it correct?